doc/content/integrations/webhooks/cloud-integrations/ubidots/ubidots-setup/_index.md

```diff
@@ -0,0 +1,99 @@
+---
+title: "Ubidots Setup"
+description: ""
+weight: 1
+---
+
+Create a function by following these instructions to prepare Ubidots setup for the integration with {{% tts %}}. This function will handle the conversion of incoming messages from JSON format to the one that is compatible with Ubidots.
+
+<!--more-->
+
+Log in to your Ubidots account and find **Devices** tab in the upper part of your dashboard. In its drop-down list, choose **Functions**.
+
+When redirected to **UbiFunctions** page, create a new function with **Create Function** button.
+
+On the left, give a **Name** to your function, select **POST** method and choose **Python 3.6** for **Runtime**.
```

Comment:
Plz add a screenshot here

Reply:
This is all covered with the screenshot at the end of this file. I put it at the end because the screenshot is containing the **HTTP Endpoint URL**, which is obtained after completing all the steps. Should I move it?